Gemfile.lock

```diff
@@ -0,0 +1,176 @@
+PATH
```

Comment:
Don't forget to remove the Gemfile.lock in the CI

Reply:
Do we need to in this case? Since CI doesn't even run with the regular Gemfile. Only the one inside the `gemfiles` folder

Reply:
Ah, so we don't need